pkg/broker/handler/processors/filter/processor_test.go

```diff
@@ -242,19 +243,23 @@ func TestFilterProcessor(t *testing.T) {
 
 	for _, tc := range cases {
 		t.Run(tc.name, func(t *testing.T) {
+			testTarget := &config.Target{
+				Name:             "target",
+				Broker:           "broker",
+				Namespace:        "ns",
+				FilterAttributes: tc.filter,
+			}
+			testTargets := memory.NewEmptyTargets()
```

Comment:
nit: `use memory.NewTargets(testTarget)`

Reply:
`NewTargets` accepts a raw `TargetsConifg` right? I didn't do that because I would have to know the key/value mapping in the test (which causes a little leaky abstraction but mostly fine). But it also turns out using `MutateBroker` doesn't introduce more code so I chose it.